src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -61,7 +61,7 @@ final class MockTestStore: Store, PPAnalyticsData {
 	var mostRecentRiskCalculationConfiguration: RiskCalculationConfiguration?
 	var forceAPITokenAuthorization = false
 	var recentTraceLocationCheckedInto: DMRecentTraceLocationCheckedInto?
-	var isSrsPrechecksEnabled = false
+	var dmIsSRSPreChecksEnabled = false
```

Comment:
Don't know if this naming is consistent with our approach

Reply:
@30mar what do you think?

Reply:
I think isSrsPrechecksEnabled is fine as it was, we use flags for dev menus without adding the dm part.

Reply:
Good point, renamed it to `isSrsPrechecksEnabled`